ng2-components/ng2-activiti-form/src/components/activiti-form.component.ts

```diff
@@ -412,12 +402,10 @@ export class ActivitiForm implements OnInit, AfterViewChecked, OnChanges {
                 .completeTaskForm(this.form.taskId, this.form.values, outcome)
                 .subscribe(
                     () => {
-                        this.formCompleted.emit(this.form);
+                        this.onTaskCompleted(this.form);
```

Comment:
No, `onTaskCompleted` just raises more than one event at the same time. It fires old `formCompleted` and also `formService.taskCompleted` events.